web/app/features/Blog/service/sanity/util.ts

```diff
@@ -1,6 +1,10 @@
+interface BlockContentItem {
+  _id: string;
+}
+
 export function filterDataToSingleItem(data = [], preview = false) {
   if (preview) {
-    return data.find(item => item._id.startsWith(`drafts.`)) || data[0];
+    return data.find((item: BlockContentItem) => item._id.startsWith(`drafts.`)) || data[0];
```

Comment:
The ideal would be set type in data parameter instead item parameter of find callback, typescript would infer the type in item.